tinygrad/utils.py

```diff
@@ -10,8 +10,8 @@ def layer_init_uniform(*x):
   return ret.astype(np.float32)
 
 def fetch(url):
-  import requests, os, hashlib
-  fp = os.path.join("/tmp", hashlib.md5(url.encode('utf-8')).hexdigest())
+  import requests, os, hashlib, tempfile
+  fp = os.path.join(tempfile.gettempdir(), hashlib.md5(url.encode('utf-8')).hexdigest())    
```

Comment:
Err, does this get the same folder every time?

Reply:
No, it gets the temp folder based on your os for example

* On RiscOS, the directory named by the Wimp$ScrapDir environment variable.
* On Windows, the directories C:\TEMP, C:\TMP, \TEMP, and \TMP, in that order.
* On Linux based distro, the directories /tmp, /var/tmp, and /usr/tmp, in that order